server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -93,17 +90,12 @@ public JettyBackedGrpcServer(
         jetty = new Server();
         jetty.addConnector(createConnector(jetty, config));
 
-        final WebAppContext context =
-                new WebAppContext(null, "/", null, null, null, new ErrorPageErrorHandler(), NO_SESSIONS);
-        try {
-            String knownFile = "/ide/index.html";
-            URL ide = JettyBackedGrpcServer.class.getResource(knownFile);
-            Resource jarContents = Resource.newResource(ide.toExternalForm().replace("!" + knownFile, "!/"));
-            context.setBaseResource(ControlledCacheResource.wrap(jarContents));
-        } catch (IOException ioException) {
-            throw new UncheckedIOException(ioException);
-        }
-        context.setInitParameter(DefaultServlet.CONTEXT_INIT + "dirAllowed", "false");
+        final ServletContextHandler context =
+                new ServletContextHandler(null, "/", null, null, null, new ErrorPageErrorHandler(), NO_SESSIONS);
+        String knownFile = "/ide/index.html";
+        URL ide = JettyBackedGrpcServer.class.getResource(knownFile);
+        String path = ide.toExternalForm().replace("!" + knownFile, "!/");
```

Comment:
probably worth a comment about "`"!/"` specifies this resource base is a jar file which should have its contents served" (but worded better maybe)

Reply:
Reverted to roughly the old code, and added a note.